Assets/MixedRealityToolkit/_Core/Definitions/Devices/DeviceInputType.cs

```diff
@@ -10,67 +10,80 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices
     /// </summary>
     public enum DeviceInputType
     {
+        //Physical Input Type       -       Recommended Axis Type
         None = 0,
         Gaze,
         Voice,
         /// <summary>
         /// 6 Dof Pointer with position and rotation.
         /// </summary>
-        SpatialPointer,
+        SpatialPointer,                     // SixDof Axis
         /// <summary>
         /// 3 Dof Pointer with only position.
         /// </summary>
-        PointerPosition,
+        PointerPosition,                    // ThreeDofPosition Axis
         /// <summary>
         /// 3 Dof Pointer with only rotation.
         /// </summary>
-        PointerRotation,
-        PointerClick,
-        ButtonPress,
-        ButtonTouch,
-        ButtonNearTouch,
-        SecondaryButtonPress,
-        SecondaryButtonTouch,
-        SecondaryButtonNearTouch,
-        Trigger,
-        TriggerTouch,
-        TriggerNearTouch,
-        TriggerPress,
+        PointerRotation,                    // ThreeDofRotation Axis
+        PointerClick,                       // Digital
+        ButtonPress,                        // Digital
+        ButtonTouch,                        // Digital
+        ButtonNearTouch,                    // Digital
+        SecondaryButtonPress,               // Digital
```

Comment:
Remove secondary input types. They're redundant.

Reply:
Agreed with this new plan.